deployvm: Showing additoinal fields in infocard

davidjumani opened the pull request:
Adds additional fields to the Infocard while creating a vm
- Pod
- Cluster
- Host
- Cpu cores
- Cpu speed
- Memory

![Screenshot from 2020-09-03 17-16-10](https://user-images.githubusercontent.com/8244774/92111020-48b29180-ee09-11ea-998f-ca7d772129ce.png)

davidjumani commented:
@blueorangutan package

blueorangutan commented:
@davidjumani a Jenkins job has been kicked to build primate packages. I'll keep you posted as I make progress.

blueorangutan commented:
@davidjumani a Jenkins job has been kicked to build primate packages. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result: :heavy_check_mark:centos :heavy_check_mark:debian :heavy_check_mark:archive.
QA: http://primate-qa.cloudstack.cloud:8080/client/pr/664 (JID-3430)

rhtyd commented:
@davidjumani I think we did not include those details to limit the height of the infocard - check on smaller screen if we scroll it well?
@utchoang can you review and test this?

davidjumani commented:
@rhtyd The user can scroll on the Infocard. The reason I added this was so that the user can see the host & compute resources allocated on the side rather than scrolling up to verify the data passed
Added images when in a smaller screen
![Screenshot from 2020-09-07 14-08-49](https://user-images.githubusercontent.com/8244774/92367188-b9acbe80-f113-11ea-946e-366682ef3233.png)
![Screenshot from 2020-09-07 14-08-59](https://user-images.githubusercontent.com/8244774/92367191-baddeb80-f113-11ea-8ef4-e0672316df39.png)

utchoang commented on src/components/view/InfoCard.vue:
```diff
@@ -280,6 +280,19 @@
           </div>
         </div>
+        <div class="resource-detail-item" v-if="resource.networks && resource.networks.length > 0">
+          <div class="resource-detail-item__label">Network(s)</div>
```
@davidjumani can you add a translation for this text?

davidjumani commented:
@blueorangutan package

blueorangutan commented:
@davidjumani a Jenkins job has been kicked to build primate packages. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result: :heavy_check_mark:centos :heavy_check_mark:debian :heavy_check_mark:archive.
QA: http://primate-qa.cloudstack.cloud:8080/client/pr/664 (JID-3439)

utchoang reviewed (approved):
LGTM! @rhtyd.
Thanks @davidjumani
